### DISCUSSION TITLE
Support Response Time for Pro Account

### DISCUSSION THREAD

**Discussion by git-no** [vercel/community#3866] [Help] [answered]
### Summary

Hello Vercel, may I ask how are the 

**Support Response Time for a Pro Account.**

I am asking because I created FIVE days ago a support request within Vercel dashboard, no response by Vercel till now.

Kind Regards
Lukas

**Comment by amyegan** [comment, marked as answer]
Hey @git-no. Tickets are processed by the order in which they are received, so response times may vary. You can find more info about support terms and wait times on the following pages:
- [Support Terms](https://vercel.com/legal/support-terms)
- [How long does it take to get a response from Vercel Support?](https://vercel.com/guides/vercel-support-queue-time)
- [Terms of Service](https://vercel.com/legal/terms)

**Reply by git-no** [reply]
Thank you.
